src/app/navbar/navbar.component.scss

```diff
@@ -79,3 +79,29 @@ mat-menu {
   color: mat.get-color-from-palette($kim-accent1-success, 4);
   transform: scale(1.25);
 }
+
+#dropdown-main {
+  padding: 14px 14px;
```

Comment:
Aren't we supposed to avoid pixels and use rems instead? It's in the [opinionated style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide), although at one point I thought Gary mentioned it may not be necessary any more. @kathy-t may know?

If rems aren't required, can you please update the style guide, so I don't keep making comments like this. If they are required, please update here.

Reply:
I was thinking about that too but wanted to make it the same padding as the notification bell and My Dockstore buttons which had 14px as padding. https://github.com/dockstore/dockstore-ui2/blob/e5d682ce2906cd65b99dc5be0b41861404244f87/src/app/navbar/navbar.component.scss#L21

If we should use rem insread, i'll change it for those as well or consistency!

Reply:
Hmm, I generally follow the style guide and use rem. I don't really see a downside to using REM because it makes the design more accessible